client/components/tinymce/plugins/simple-payments/dialog/index.jsx

```diff
@@ -250,6 +256,11 @@ class SimplePaymentsDialog extends Component {
 		} );
 	};
 
+	addNewPaymentButton = () => {
+		this.setState( { activeTab: 'form', editedPaymentId: null } );
+		this.formStateController.resetFields( this.getInitialFormFields( null ) );
```

Comment:
This method is identical to calling `editPaymentButton( null )`. What do you think about renaming `editPaymentButton` to `showButtonForm` and calling `showButtonForm( null )` instead of having the separate `addNewPaymentButton` method? That's what the changes in #16960 do.